app/javascript/components/SignUp.jsx

```diff
@@ -23,35 +32,70 @@ class SignUp extends Component {
         this.setState({[name]: value});
     }
 
+    onChangeBirthdateOption(e) {
+        e.preventDefault();
+        const name = e.target.name;
+        const value = e.target.value;
+        this.setState({ [name]: value });
+    }
+
     handleSubmit(e){
         e.preventDefault();
 
         this.formSubmit(e.target);
     };
 
-    async formSubmit(formData){
-        var data = new FormData(formData);
+    async formSubmit(){
         await fetch('/signup_user', {
-            Accept: 'application/json',
+            headers: {
```

Comment:
We should create a utils folder for api call functions. The more api calls we have to make, the messier this will get. ie. https://github.com/carlosarias1992/Fakebook/blob/61de622171724ea1efc366049ace95150ebfd081/frontend/util/session_api_util.js#L1

Reply:
By the way, you don't have to use ajax, but that's an example of the util function for signing up users.